dtm/remote.c

```diff
@@ -244,8 +244,8 @@ static int rem_rpc_deliver(struct m0_rpc_machine *mach,
 	return 0;
 }
 
-static void rem_rpc_redo_replied(struct m0_rpc_item *item)
-{}
+//static void rem_rpc_redo_replied(struct m0_rpc_item *item)
```

Comment:
Removed the unused code, or use M0_UNUSED.

Reply:
accordingly added the M0_UNUSED macro